prime-angular-frontend/src/app/modules/adjudication/shared/components/enrollee-table/enrollee-table.component.ts

```diff
@@ -60,6 +60,10 @@ export class EnrolleeTableComponent implements OnInit, OnChanges {
   public EnrolmentStatus = EnrolmentStatusEnum;
   public Role = Role;
   public readonly PAPER_ENROLLEE_GPID_PREFIX = PAPER_ENROLLEE_GPID_PREFIX;
+  public readonly paperEnrolmentStatusMap = new Map<number, string>([
```

Comment:
Doesn't really reflect this line of code, but the chosen solution for this ticket is as follows,

"Add new status types of IncompletePaperEnrolment and CompletePaperEnrolment, and update the UI to show “Incomplete” and “Complete”."

Not sure if there was other conversations to change this but that requires adding those types in the back end... etc
That is why this ticket was scored at 3 pts, just doing it in the front end is 0.5 of work.

Reply:
Yes I had conversations with James, Martin, and Jason about that. To add the status we would have to make the change in all the places where Editable and Under Review were used and that would be very risky. So we decided to just map the front end.

I forgot to update the ticket on Jira